common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -7827,7 +7827,7 @@
   },
   "_LEGACY_ERROR_TEMP_3055" : {
```

Comment:
Nit: If we're already touching this _LEGACY, could we just name it? (is this an internal error??)

Reply:
I opened the starter task https://issues.apache.org/jira/browse/SPARK-49549 @mitkedb FYI.